### ISSUE TITLE
Remove "clear cookies" option from default menu

### ISSUE THREAD

**Issue by aveao** [mbnuqw/sidebery#1336]
## Description

![image](https://github.com/mbnuqw/sidebery/assets/6755762/a76b0fff-e742-47ab-9003-9fbac2587377)

I tried to close ~200 tabs and ended up logging out of pretty much everywhere instead.

This could be made nicer by:
- Moving the "Clear Cookies" button elsewhere so that it's not next to "Close Tab"
- Asking for confirmation when it is used on many tabs, similar to "warn on trying to close multiple tabs".
- Generally making the options in the context menu configurable, so that one can choose to enable such a nuclear option in their context menu only if they want to.

**Comment by albino1** [comment]
I agree that for the vast majority of people clearing cookies is a pretty uncommonly used button to get such a prominent place, it should probably be opt in and not opt out to even have it visible at all.

FWIW, I just removed it from my context menu entirely by going to:

Settings -> Menu Editor -> Tabs

**Comment by aveao** [comment]
> I agree that for the vast majority of people clearing cookies is a pretty uncommonly used button to get such a prominent place, it should probably be opt in and not opt out to even have it visible at all.

Indeed. I'm not entirely sure what the feature's purpose is as it doesn't even wipe local storage, so many sites will still be able to keep you logged in or if they wish, track you.

> FWIW, I just removed it from my context menu entirely by going to:
> 
> Settings -> Menu Editor -> Tabs

Thanks! I must've missed it, the settings are pretty intense to navigate.

**Comment by saveman71** [comment]
I just cleared like 300 tabs worth of cookies :sweat_smile: 

I believe this is the third time I was hit by this, this is nice that this is now removed as a default option but I guess having the extension installed for a while my settings were preserved from this update. I have now removed it from my config (discovering that the menu is configurable is a nice learning from that though ;))

It's a welcomed change, but I still believe a confirmation should be added to this (relatively) "dangerous" action when it'll impact more than 1 tab (domain?).

Should I create a new issue for tracking? I can't find another issue referencing this, and now that it's not a default it's most likely not impacting as many users (but most users having an older config if I'm understanding well how config changes are applied).
